src/main/java/core/basesyntax/dao/impl/UserDaoImpl.java

```diff
@@ -3,30 +3,73 @@
 import core.basesyntax.dao.UserDao;
 import core.basesyntax.model.User;
 import java.util.List;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
 
 public class UserDaoImpl extends AbstractDao implements UserDao {
     public UserDaoImpl(SessionFactory sessionFactory) {
         super(sessionFactory);
     }
 
     @Override
-    public User create(User entity) {
-        return null;
+    public User create(User user) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(user);
+            transaction.commit();
+            return user;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Could not save user with id " + user.getId(), e);
```

Comment:
does user have id at this moment? 
```suggestion
            throw new RuntimeException("Could not save user with id " + user, e);
```